config/settings.yml

```diff
@@ -142,6 +142,10 @@ people:
   default_sort: name
   abos: true
   totp_drift: 15
+  gender_custom:
+    enabled: false
+  gender_binary:
+    enabled: true
```

Comment:
This means that it is possible to have both binary and custom gender-selectors, right? Is that intended? 

It feels like a workaround for the `FeatureGate` missing a proper `else`. If that is the case, I can look into that.